server/lib/budget.js

```diff
@@ -8,19 +8,45 @@ import { getFxRate } from './currency';
 const { CREDIT, DEBIT } = TransactionTypes;
 const { PROCESSING, SCHEDULED_FOR_PAYMENT } = expenseStatus;
 
+async function computeSumFromResults(results, currency) {
```

Comment:
Maybe name something related to currency conversion: `sumTransactionsInCurrency`.